Terraform crash on import:

I get this crash attempting to import storage buckets in the google provider for terraform:

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
COMP0307:gcp james.mullan$ terraform import module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1 failbox-testbucket-1
module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1: Importing from ID "failbox-testbucket-1"...
module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1: Import complete!
Imported google_storage_bucket (ID: failbox-testbucket-1)
module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1: Refreshing state... (ID: failbox-testbucket-1)

Error: module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1 (import id: failbox-testbucket-1): 1 error(s) occurred:

import module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1 result: failbox-testbucket-1: google_storage_bucket.failbox-testbucket-1: unexpected EOF
panic: runtime error: invalid memory address or nil pointer dereference
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1ed9524]
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4:
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: goroutine 25 [running]:
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/google.flattenBucketLifecycleRuleCondition(0xc4202124d0, 0xc4204bb740)
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/google/resource_storage_bucket.go:552 +0x1d4
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/google.flattenBucketLifecycle(0xc4205ab3b0, 0x23e44a0, 0xa, 0x2005ea0)
2018-01-22T14:46:36.576-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/google/resource_storage_bucket.go:534 +0x2de
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/google.resourceStorageBucketRead(0xc4201b11f0, 0x20f5520, 0xc420441860, 0xc4201b11f0, 0x0)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/google/resource_storage_bucket.go:398 +0xa48
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema.(*Resource).Refresh(0xc420494e70, 0xc420164500, 0x20f5520, 0xc420441860, 0xc420439400, 0xc4203a0b01, 0x80000000018)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:354 +0x17e
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema.(*Provider).Refresh(0xc420495110, 0xc4201644b0, 0xc420164500, 0x34416c8, 0x0, 0x18)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:308 +0x9a
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/plugin.(*ResourceProviderServer).Refresh(0xc4203f69a0, 0xc42017b810, 0xc42017b9b0, 0x0, 0x0)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /Users/james.mullen/go/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/plugin/resource_provider.go:549 +0x4e
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: reflect.Value.call(0xc4203ba840, 0xc42016a498, 0x13, 0x23dd979, 0x4, 0xc420421f20, 0x3, 0x3, 0x0, 0x0, ...)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /usr/local/Cellar/go/1.9.2/libexec/src/reflect/value.go:434 +0x905
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: reflect.Value.Call(0xc4203ba840, 0xc42016a498, 0x13, 0xc4203fcf20, 0x3, 0x3, 0x13d37fa, 0x1, 0x0)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /usr/local/Cellar/go/1.9.2/libexec/src/reflect/value.go:302 +0xa4
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: net/rpc.(*service).call(0xc4203ea440, 0xc420164140, 0xc4203ec518, 0xc42042a680, 0xc4204ea940, 0x1fc0260, 0xc42017b810, 0x16, 0x1fc02a0, 0xc42017b9b0, ...)
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /usr/local/Cellar/go/1.9.2/libexec/src/net/rpc/server.go:381 +0x142
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: created by net/rpc.(*Server).ServeCodec
2018-01-22T14:46:36.577-0500 [DEBUG] plugin.terraform-provider-google_v1.5.0_x4: /usr/local/Cellar/go/1.9.2/libexec/src/net/rpc/server.go:475 +0x36b
2018/01/22 14:46:36 [ERROR] root.tapad-cross-platform-test-module: eval: *terraform.EvalRefresh, err: google_storage_bucket.failbox-testbucket-1: unexpected EOF
2018/01/22 14:46:36 [ERROR] root.tapad-cross-platform-test-module: eval: *terraform.EvalSequence, err: google_storage_bucket.failbox-testbucket-1: unexpected EOF
2018/01/22 14:46:36 [TRACE] [walkImport] Exiting eval tree: import module.tapad-cross-platform-test-module.google_storage_bucket.failbox-testbucket-1 result: failbox-testbucket-1
2018/01/22 14:46:36 [TRACE] dag/walk: upstream errored, not walking "provider.google (close)"
2018/01/22 14:46:36 [DEBUG] plugin: waiting for all plugin processes to complete...
2018-01-22T14:46:36.581-0500 [WARN ] plugin: error closing client during Kill: err="connection is shut down"
2018-01-22T14:46:36.581-0500 [DEBUG] plugin: plugin process exited: path=/Users/james.mullen/work/terraform/gcp/.terraform/plugins/darwin_amd64/terraform-provider-google_v1.5.0_x4

--- a/google/resource_storage_bucket.go
+++ b/google/resource_storage_bucket.go
@@ -546,13 +546,16 @@ func flattenBucketLifecycleRuleAction(action *storage.BucketLifecycleRuleAction)
 }
 
 func flattenBucketLifecycleRuleCondition(condition *storage.BucketLifecycleRuleCondition) map[string]interface{} {
-	return map[string]interface{}{
+	ruleCondition := map[string]interface{}{
 		"age":                   int(condition.Age),
 		"created_before":        condition.CreatedBefore,
-		"is_live":               *condition.IsLive,
 		"matches_storage_class": convertStringArrToInterface(condition.MatchesStorageClass),
 		"num_newer_versions":    int(condition.NumNewerVersions),
 	}
+	if condition.IsLive != nil {
+		ruleCondition["is_live"] = *condition.IsLive
+	}
+	return ruleCondition
 }
 
 func resourceGCSBucketLifecycleCreateOrUpdate(d *schema.ResourceData, sb *storage.Bucket) error {

